README.md

```diff
@@ -18,7 +18,7 @@
 The [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) mission
 is to make [nixpkgs](https://github.com/nixos/nixpkgs) the most
 up-to-date repository of software in the world by the most ridiculous
-margin possible.
+margin possible. So far, [we succeed](hhttps://repology.org/repositories/graphs).
```

Comment:
```suggestion
margin possible. [Check how we are doing so far!](hhttps://repology.org/repositories/graphs).
```

Reply:
Maybe this will be more pragmatic / cool:

```suggestion
margin possible. [Here's how we are doing so far](https://repology.org/repositories/graphs).
```